### PR TITLE
disable loading widgets

### PR DESCRIPTION
**Pull request by willmcgugan**
Fixes https://github.com/Textualize/textual/issues/5266

### CODE REVIEW

File: src/textual/widgets/_loading_indicator.py [hunk 1]
```diff
@@ -56,6 +58,12 @@ def _on_mount(self, _: Mount) -> None:
         self._start_time = time()
         self.auto_refresh = 1 / 16
 
+    @on(InputEvent)
+    def on_input(self, event: InputEvent) -> None:
+        """Prevent all input events from bubbling, thus disabling widgets in a loading state."""
+        event.stop()
```
Comment (willmcgugan) [inline src/textual/widgets/_loading_indicator.py:64]: @darrenburns Should we do app.bell here? Seems just the error condition one might use it for.

Comment (TomJGooding) [reply src/textual/widgets/_loading_indicator.py:64]: I'm not Darren, but does Textual use the terminal bell in any other contexts? Personally I don't think trying to interact with a loading widget should make a fuss about it.

Comment (willmcgugan) [reply src/textual/widgets/_loading_indicator.py:64]: We play the bell when clicking on disabled footer keys, and when input is restricted. So there is precedence.

I suspect bell makes more sense in disabled widgets.

**Comment by willmcgugan** [comment]
#executivedecision

**Pull request merged**